### DISCUSSION TITLE
Android 源码分析开篇 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#212] [Announcements]
https://mazhuang.org/2017/09/23/rtfsc-android-start/

**Comment by Leejunxiang** [comment]
加油吧

**Comment by mzlogin** [comment]
谢谢~看了下你的博客，你好小啊，好厉害！:smile:

> @Leejunxiang
> 加油吧

**Comment by Leejunxiang** [comment]
> @mzlogin
> 谢谢~看了下你的博客，你好小啊，好厉害！:smile:
> 
> > @Leejunxiang
> > 加油吧
> 
> 

我的一个朋友开了一个博客,看起来不错,于是我就尝试自己也搞一个.尔后在知乎看到了你的那个模版,再找了些教程,也勉强搭起来了这样一个blog

**Comment by mzlogin** [comment]
厉害~长江后浪推前浪 :+1:

> @Leejunxiang
> > ...
> 
> 我的一个朋友开了一个博客,看起来不错,于是我就尝试自己也搞一个.尔后在知乎看到了你的那个模版,再找了些教程,也勉强搭起来了这样一个blog

**Comment by le773** [comment]
期待更新

**Comment by mzlogin** [comment]
感谢关注~感觉到了责任😬

> @le773
> 期待更新

**Comment by 001bufanbin** [comment]
楼主，使用你的模板创建的blog，disqus被墙了打不开，怎么注册啊？，现在有了gitment和gitalk了。

**Comment by mzlogin** [comment]
> @001bufanbin
> 楼主，使用你的模板创建的blog，disqus被墙了打不开，怎么注册啊？，现在有了gitment和gitalk了。

不需要都注册呢，这仨你挑其中一个就可以了，可以就使用 gitment 或 gitalk 中的一个，然后将 lazy_load_disqus 改为 false。如果一定要使用 disqus，那就得你想办法翻了。

**Comment by ctwillson** [comment]
**评论是否可以做成微信 QQ登录的。。。**

**Comment by mzlogin** [comment]
> @ctwillson
> **评论是否可以做成微信 QQ登录的。。。**
> 

评论都是用的第三方做的插件，当前使用的是 Gitalk，是利用 GitHub 的 Issue 存储评论内容，估计按现有架构无法实现微信/QQ 登录，不过你可以去 <https://github.com/gitalk/gitalk> 提个 Issue 碰碰运气……当然你也可以换成支持微信/QQ 登录的第三方评论框，比如 [来必力](http://www.laibili.com.cn/)。

**Comment by fengerzh** [comment]
我看你又换成gitment了，gitalk和gitment之间有什么关系吗？

**Comment by mzlogin** [comment]
> @fengerzh
> 我看你又换成gitment了，gitalk和gitment之间有什么关系吗？

我现在使用的是 gitalk。它俩的模式和原理是一模一样的，先有 gitment 后有 gitalk，早期 gitalk 应该是仿的 gitment。不过 gitment 在今年四月之后就没怎么更新了，而 gitalk 在更新上更活跃。

它俩现在主要的区别是：

1. gitment 在样式上更贴近 GitHub 原生界面，简洁流，而 gitalk 界面样式更丰富一点；
2. 发布新文章之后，gitalk 不用手动点击按钮触发初始化评论框，登录 GitHub 账号后访问一次页面即可。

它俩的数据源可以共用，自由切换，在 gitalk 的配置里设置 `labels: ['gitment']` 即可。

**Comment by ophina** [comment]
无意中看到楼主的博客，fork一下，已关注，然后加友链了，开源精神
